src/javascript/app/pages/form/form-component.jsx

```diff
@@ -344,7 +344,7 @@ export const FormComponent = () => {
                                                     currency_list={currency_list}
                                                     currency={currency}
                                                     onUpdate={updateFormField}
-                                                    elementId='currency'
+                                                    element_id='currency'
```

Comment:
`elementId` to `element_id` in `CurrencyDropdown` as well.

Reply:
same for this I kept it elementId